### PR TITLE
Round out data documentation

### PR DESCRIPTION
**Pull request by khmoran**

### CODE REVIEW

File: data-serving/README.md [hunk 1]
```diff
@@ -1,18 +1,78 @@
 
 # All things data
 
-## Case schema updates
-### What needs to be changed & where
+[toc]
```
Comment (axmb) [inline data-serving/README.md:4]: I don't think this works out of the box for GitHub. :-(

File: data-serving/README.md [hunk 2]
```diff
@@ -1,18 +1,78 @@
...
+
+## Database design
+
+G.h case data, as well as the source, user, and session data for the curator portal, is stored in a MongoDB database.
+
+We have multiple instances of MongoDB, ranging from local instances for development, to dev (for
+http://dev-curator.ghdsi.org/), to prod prod (for http://dev-curator.ghdsi.org/).
```
Comment (axmb) [inline data-serving/README.md:11]: Extra `prod`? And prod URL shouldn't have `dev-`.

File: data-serving/README.md [hunk 3]
```diff
@@ -1,18 +1,78 @@
...
+We store past revisions of `case` documents in the `caserevisions` collection. These revisions are made in the
+application layer when a case is updated; we follow the
+[MongoDB Document Version Pattern](https://www.mongodb.com/blog/post/building-with-patterns-the-document-versioning-pattern).
+
+A `caserevision` document has a `case` field containing a snapshot of the `case` at a given revision. The collection
+indexes the id of the `case` and its revision for quick lookups.
+
+### Charts
+
+We use MongoDB to generate charts that surfaced in the curator portal. These are available through the prod MongoDB
```
Comment (axmb) [inline data-serving/README.md:39]: Perhaps drop a link to the charts here (the portal and/or the Atlas console).

File: data-serving/README.md (hunk 2 above)
Comment (khmoran) [reply data-serving/README.md]: sorta seems like I broke

File: data-serving/README.md (hunk 1 above)
Comment (khmoran) [reply data-serving/README.md]: :'( that's so sad. it worked nicely in my VScode md preview so I just assumed. I'll just remove

**Pull request merged**